tests/javascript/frameworks/dojo/dojo-1.0.3.js

```diff
@@ -2808,52 +2808,6 @@ if(!dojo._hasResource["dojo._base.window"]){ //_hasResource checks added by buil
 dojo._hasResource["dojo._base.window"] = true;
 dojo.provide("dojo._base.window");
 
-dojo._gearsObject = function(){
```

Comment:
guess that can be reverted, as it's changing an external library, that's only used for tests...